sda/cmd/download/swagger_v1.yml

```diff
@@ -0,0 +1,273 @@
+openapi: 3.0.3
+info:
+  title: SDA file download API
+  version: "1.0"
+paths:
+  /file:
+    get:
+      description: Returns the requested file to the user
+      parameters:
+        - in: header
+          description: "Public key used to re-encrypt the file header with"
+          name: public_key
+          required: true
+          schema:
+            type: string
+        - in: header
+          description: "The JWT used to authenticate the user"
+          name: token
+          required: true
+          schema:
+            type: string
+        - in: query
+          name: dataset
+          schema:
+            type: string
+          required: true
+          description: Stable ID of the dataset
+        - in: query
+          name: fileId
+          schema:
+            type: string
+          required: false
+          description: Stable ID of the file to download
+        - in: query
+          name: filePath
+          schema:
+            type: string
+          required: false
+          description: File path of the file to download
+        - in: query
+          name: end
+          required: false
+          schema:
+            default: 0
+            minimum: 0
+            type: integer
+        - in: query
+          name: start
+          required: false
+          schema:
+            default: 0
+            minimum: 0
+            type: integer
+      responses:
+        "200":
+          content:
```

Comment:
```suggestion
         headers:
           'Content-Disposition: attachment; filename="filePath"'
          content:
```

?

Reply:
Not really, if you request a file by stable ID that should be the filename in the header.

Reply:
Fair, would this be useful enough that we should put the basename there or shall we skip?

Reply:
Let's skip this one. It's not likely that someone will try to download a file this way using a browser anyways.